packages/block-editor/src/layouts/grid.js

```diff
@@ -294,10 +335,10 @@ function GridLayoutColumnsAndRowsControl( {
 								onChange={ ( value ) =>
 									onChange( {
 										...layout,
-										columnCount: value,
+										columnCount: parseInt( value, 10 ),
```

Comment:
`RangeControl` provides `number | undefined` to `onChange` so this `parseInt` is unnecessary.

Reply:
Oh I meant to do that to the NumberControl above 🤦